cypress/integration/changeSignalAfterSubmit.spec.js

```diff
@@ -77,7 +76,6 @@ describe('Change signal after submit', () => {
       cy.contains(Cypress.env('description')).should('be.visible');
 
       // Check signal data
-      cy.get(SIGNAL_DETAILS.stadsdeel).should('have.text', 'Stadsdeel: Nieuw-West').should('be.visible');
```

Comment:
All checks on 'Stadsdelen' should stay. You need data in the backend to test this.